packages/driver/src/cross-origin/cypress.ts

```diff
@@ -207,7 +206,7 @@ const attachToWindow = (autWindow: Window) => {
 
       cy.isStable(false, 'beforeunload')
 
-      cy.Cookies.setInitial()
+      // NOTE: we intentionally do not set the cy.Cookies.setInitial() inside the spec bridge as we are not doing full injection and this leads to cookie side effects
```

Comment:
What side effects was this causing? Is this a separate issue from "same origin spec bridges" or was it exposed during these code changes?

Reply:
Main issue is sometimes setting a cookie on the domain that isn't cleared in firefox that we don't even need because we don't do full injection. I would say it was always an issue and was exposed more so with these changes.

example within firefox _before_:
spec bridge was creating a `__cypress.initial` cookie on `foobar.com` domain (not `.foobar.com`)
AUT is at `www.foobar.com` and makes requests. `__cypress.initial` cookie isn't attached because it doesn't fit the domain policy (`foobar.com` vs `.foobar.com`)
![Screen Shot 2022-10-04 at 10 58 22 AM](https://user-images.githubusercontent.com/3980464/193854265-eaa8f978-2798-47d5-8af6-c91bec268a5e.png)

example within firefox _after_:
spec bridge was creating a `__cypress.initial` cookie on `www.foobar.com` domain (`www.foobar.com`)
AUT is at `www.foobar.com` and makes requests. `__cypress.initial` cookie is attached and is never cleared
![Screen Shot 2022-10-04 at 11 08 03 AM](https://user-images.githubusercontent.com/3980464/193856355-95ae64b5-4663-4ad6-b675-b1aec8886a24.png)

Not sure if this is an underlying issue with `js-cookie`, but the domain that is set in firefox for the cookie is different than that of chromium browsers. We don't see it in chromium browsers because the domain set in the spec bridge is localhost
<img width="1778" alt="Screen Shot 2022-10-04 at 10 44 17 AM" src="https://user-images.githubusercontent.com/3980464/193856471-e1b3baa8-d1b0-4b5a-86b3-c6e13cb4254e.png">